ngraph/frontend/tensorflow/include/tensorflow_frontend/graph_iterator.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tensorflow_frontend/decoder.hpp>
+
+namespace ngraph {
+namespace frontend {
+/// Abstract representation for an input model graph that gives nodes in topologically sorted order
+class GraphIterator {
+public:
+    /// Get a number of operation nodes in the graph
```

Comment:
I guess we should add /// \brief  everywhere

Reply:
Added. Thanks.